tket/src/Ops/ClassicalOps.cpp

```diff
@@ -239,29 +240,38 @@ std::vector<bool> ClassicalTransformOp::eval(const std::vector<bool> &x) const {
 }
 
 WASMOp::WASMOp(
-    unsigned _n, std::vector<unsigned> _ni_vec, std::vector<unsigned> _no_vec,
-    const std::string &_func_name, const std::string &_wasm_uid)
+    unsigned _n, unsigned _ww_n, std::vector<unsigned> _width_i_parameter,
+    std::vector<unsigned> _width_o_parameter, const std::string &_func_name,
+    const std::string &_wasm_file_uid)
     : ClassicalOp(
           OpType::WASM,
           std::accumulate(
-              _ni_vec.begin(), _ni_vec.end(), decltype(_ni_vec)::value_type(0)),
+              _width_i_parameter.begin(), _width_i_parameter.end(),
+              (unsigned)0),
           0,
           std::accumulate(
-              _no_vec.begin(), _no_vec.end(), decltype(_no_vec)::value_type(0)),
+              _width_o_parameter.begin(), _width_o_parameter.end(),
+              (unsigned)0),
```

Comment:
Is the cast needed?

Reply:
No, removed in @cqc-melf
[remove cast from sum in parameters](https://github.com/CQCL/tket/pull/743/commits/d27656e18fba7fccb2a6dd88f4014111ccf06214)
[d27656e](https://github.com/CQCL/tket/pull/743/commits/d27656e18fba7fccb2a6dd88f4014111ccf06214)